eclass/qmake-utils.eclass

```diff
@@ -102,28 +102,10 @@ eqmake5() {
 
 	ebegin "Running qmake"
 
-	"$(qt5_get_bindir)"/qmake \
-		-makefile \
-		QMAKE_AR="$(tc-getAR) cqs" \
-		QMAKE_CC="$(tc-getCC)" \
-		QMAKE_LINK_C="$(tc-getCC)" \
-		QMAKE_LINK_C_SHLIB="$(tc-getCC)" \
-		QMAKE_CXX="$(tc-getCXX)" \
-		QMAKE_LINK="$(tc-getCXX)" \
-		QMAKE_LINK_SHLIB="$(tc-getCXX)" \
-		QMAKE_OBJCOPY="$(tc-getOBJCOPY)" \
-		QMAKE_RANLIB= \
-		QMAKE_STRIP= \
-		QMAKE_CFLAGS="${CFLAGS}" \
-		QMAKE_CFLAGS_RELEASE= \
-		QMAKE_CFLAGS_DEBUG= \
-		QMAKE_CXXFLAGS="${CXXFLAGS}" \
-		QMAKE_CXXFLAGS_RELEASE= \
-		QMAKE_CXXFLAGS_DEBUG= \
-		QMAKE_LFLAGS="${LDFLAGS}" \
-		QMAKE_LFLAGS_RELEASE= \
-		QMAKE_LFLAGS_DEBUG= \
-		"$@"
+	local -a args
+	mapfile -t args <<<"$(qt5_get_qmake_args)"
```

Comment:
hmm can we use `mapfile -t`? I think it was added in bash-5.0 or so and we still need to support 4.2 for EAPIs < 8.

Reply:
mapfile was added in bash-4.0 and it support -t from the start, we're good

Reply:
great, then +1 from me, thanks!

Reply:
Thanks!